_source/_posts/2022-04-05-state-of-ffi-java.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: blog_post
+title: "Does Java finally have a secure alternative to JNI?"
```

Comment:
```suggestion
title: "Can the Foreign Function Interface Improve on JNI for Java?"
```
this scores best out of a bunch of variations, and does better without the acronym. I'll add the acronym in the text

Reply:
but that title doesn't sound very existing as a Java Dev, IMO, do you mind if I try something else?

Reply:
New title works great. Thanks.